var/da/da_gpsref/da_gpsref.f90

```diff
@@ -15,7 +15,7 @@ module da_gpsref
 ! t_iwabuchi END
       anal_type_verify,fails_error_max,pseudo_err,pseudo_x, pseudo_y, stdout, &
       use_gpsrefobs, gpsref_thinning, pseudo_z,pseudo_val,max_error_ref, pseudo, &
-      jts, jte,its,ite, npres_print, pptop
+      jts, jte,its,ite, npres_print, pptop, write_rej_obs_conv
```

Comment:
@jamiebresch 
Jamie,
Most of the other files, the option was part of the "use only", and then (for example) write_rej_obs_conv was used in an if test surrounding a write statement. I do not see a write statement in this file that is modified.

Reply:
@davegill Good job reviewing 49 files! The corresponding file for this da_gpsref.f90 is da_check_max_iv_gpsref.inc. The diff is right above this one.